lib/engine/step/g_18_cz/draft.rb

```diff
@@ -51,6 +51,11 @@ def finished?
         def actions(entity)
           return [] if finished?
 
+          unless @companies.find { |c| current_entity.cash < min_bid(c) }.nil?
```

Comment:
It'd be better doing this as part of process_bid/process_pass (the duplicated code should be removed)

Reply:
Could you help me understand, are `process_$action` called when those buttons are clicked?

(my limited understanding is we return an empty array of actions so the player is given no valid actions and their turn gets skipped)

Reply:
Yeah they're what are called.
We've strayed away from doing state changes in the actions method unless there's no other choice as it should really be stateless.

Reply:
if @companies.any? { |c| ... }

Reply:
Wouldn't this need to be in `actions()` since it is autopassing the player?
(i.e. it can't fire on the `process_` calls, since they won't be called)

Reply:
waiting for this to be cleaned up

Reply:
it's probably good enough doing it this way. otherwise it'd need a full rewrite